print-eap.c

```diff
@@ -148,56 +148,36 @@ static const struct tok eap_aka_subtype_values[] = {
  */
 void
 eap_print(netdissect_options *ndo,
-          const u_char *cp,
+          const u_char *tptr,
```

Comment:
Why changing `cp` to `tptr`?
in eapol_print(), you use `cp`.
`tptr` should be reserved as temporary copy (if needed?) in a function body, not parameter.

Reply:
Agree with your comments, Addressed. Thanks